Pipfile

```diff
@@ -33,6 +32,7 @@ xlsxwriter = "==1.0.2"
 retrying = "==1.3.3"
 sdc-rabbit = "==1.2.1"
 alembic = "==0.9.6"
+pycryptodome = "*"
```

Comment:
Should this be bound to a specific version?

Reply:
Personally I think all the pinning we do is pointless because the pipfile.lock does all the pinning but I am happy to pin it for consistency (https://github.com/pypa/pipenv/issues/954)